app/assets/javascripts/discourse/tests/integration/components/site-header-test.js

```diff
@@ -49,4 +49,38 @@ module("Integration | Component | site-header", function (hooks) {
     // Click anywhere
     await click("header.d-header");
   });
+
+  test("user avatar is highlighted when the user receives the first notification", async function (assert) {
+    this.currentUser.set("all_unread_notifications", 1);
+    this.currentUser.set("redesigned_user_menu_enabled", true);
+    this.currentUser.set("read_first_notification", false);
```

Comment:
I think we're trying to avoid `setProperties` because soon we'll be able to just do `this.currentUser.whatever = blah` and it'll be easier to migrate `set` than `setProperties`.

Reply:
:ok_hand: